# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/linux24/89-mips-abi-defines.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- linux/include/linux/elf.h.orig	Fri Jan 17 15:33:32 2003
+++ linux/include/linux/elf.h	Fri Jan 17 15:32:31 2003
@@ -38,6 +38,8 @@
 #define EF_MIPS_PIC       0x00000002
 #define EF_MIPS_CPIC      0x00000004
 #define EF_MIPS_ARCH      0xf0000000
+#define EF_MIPS_ABI2      0x00000020
+#define EF_MIPS_ABI       0x0000f000
 
 /* These constants define the different elf file types */
 #define ET_NONE   0
